lib/active_record_shards/sharded_model.rb

```diff
@@ -1,5 +1,15 @@
 module ActiveRecordShards
+  module Ext
+    module ShardedModel
+      def is_sharded?
+        true
+      end
+    end
+  end
+
   class ShardedModel < ActiveRecord::Base
     self.abstract_class = true
+
+    extend ActiveRecordShards::Ext::ShardedModel
```

Comment:
Why is this better than `def self.is_sharded?` ?

Reply:
You mean why am I extending a module over just defining a method?

Reply:
Yeah. that’s what I mean.

Reply:
I need the module to extend into various ActiveRecord classes where I can't easily change the superclass (models from gems). As I need a module I might as well reuse the module here to keep it slightly more dry.